src/components/common/Map/Map.stories.ts

```
@@ -0,0 +1,21 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import Map from '@/components/common/Map/Map';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)